encryption.md

```diff
@@ -34,6 +34,12 @@ TODO: Include language about safe nonce considerations.
 
 When the sender wishes to remain anonymous, they should identify themselves as a sender using a newly minted DID that's never been seen before. When the sender is done with the interaction, they can abandon further use of that DID. 
 
+## Media Types
+
+The media type of the envelope is included in the `typ` [property](https://tools.ietf.org/html/rfc7516#section-4.1.11) and the media type of the payload is included in the `cty` [property](https://tools.ietf.org/html/rfc7516#section-4.1.12). The [message types](#message-types) section provides general discussion of the media types.
```

Comment:
Can we switch this to normative language? This looks good otherwise

Reply:
@kdenhartog I've now added MUST include 'typ' and 'cty' to both the JWE and JWS sections.